compiler/luci/pass/src/RemoveUnnecessaryTransposeNetPass.cpp

```diff
@@ -48,6 +82,17 @@ class TaggedShapeAnalyzer final
 
   bool verify_tag() const;
 
+private:
+  const luci::CircleNode *_in = nullptr;
+  const luci::CircleTranspose *_front_transpose = nullptr;
+  const luci::CircleReshape *_mid_reshape = nullptr;
+  const luci::CircleTranspose *_back_transpose = nullptr;
+
+  std::vector<int32_t> _in_shape;
+  std::vector<int32_t> _front_perm;
+  std::vector<int32_t> _mid_shape;
+  std::vector<int32_t> _back_perm;
```

Comment:
this member will be used in other member functions. 

I'm planning to change other method like this - https://github.com/Samsung/ONE/pull/14150/files#r1784062781.